### PR TITLE
Add blockchain Flare

### PR DESCRIPTION
**Pull request by sumxu96**

### CODE REVIEW

**Review by zack-winchell-cb — APPROVED** [review]

File: .github/workflows/salus-scan.yml [hunk 1]
```diff
@@ -10,3 +10,5 @@ jobs:
     - name: Salus Scan
       id: salus_scan
       uses: federacy/scan-action@0.1.4
+      with:
+        active_scanners: "\n  - PatternSearch\n  - Semgrep\n  - Trufflehog\n  - YarnAudit"
```
Comment (sumxu96) [inline .github/workflows/salus-scan.yml:14]: Needed to fix Salus issue

**Review by qodesmith — APPROVED** [review]

File: .github/workflows/salus-scan.yml (hunk 1 above)
Comment (Edgarruiz85) [reply .github/workflows/salus-scan.yml:14]: sum/add-flare

Comment (LottisPottispoo) [reply .github/workflows/salus-scan.yml:14]: Rewiew

**Review by Billiej88 — COMMENTED** [review]
sum/add-flare

**Pull request merged**